person.rb

```diff
@@ -1,4 +1,5 @@
-class Person
+require '.\nameable'
```

Comment:
- [ ] It would be better if you use `require_relative './nameable'` to prevent any error :+1:

Reply:
Once again, it would be good if you please kindly at least bother to clone my repo and run it, you will come to know that it does not give error if you use backslash with require

Reply:
@zunairkhan811 I clone the project as you are saying but I got an error. after I refactored your code but using `require_relative './..'` then it works